hicommonwealth/commonwealth
# #5561 - Create Icon Only Button

This PR completes work described in issue #5561, which asks for the creation of a new button that contains only an icon. 

The designs can be found [here](https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?node-id=27%3A1879&mode=dev).

## Link to Issue
Closes: #5561 

## Description of Changes
- Created CWIconButton with the designs described in the link above

## Test Plan
- navigate to the `/components` page ad scroll to the Icon Buttons section
- check that the styles match those in the figma file linked to above 

## Other Considerations
- is a blocker for #5477
